solutions/instances/main.tf

```diff
@@ -149,7 +149,41 @@ module "scc" {
 }
 
 #######################################################################################################################
-# SCC WP
+# SCC Attachment
+#######################################################################################################################
+
+# Data source to account settings
+data "ibm_iam_account_settings" "iam_account_settings" {}
+
+module "create_profile_attachment" {
+  source                 = "terraform-ibm-modules/scc/ibm//modules/attachment"
+  version                = "1.4.2"
+  for_each               = toset(var.profile_attachments)
+  profile_name           = each.key
+  profile_version        = "latest"
+  scc_instance_id        = module.scc.guid
+  attachment_name        = "${var.prefix}-scc-attachment-${each.value + 1}"
```

Comment:
Can we call it something more meaningful? The name can have spaces I believe. Perhaps `"${each.value} daily full account attachment"`

No need to include the prefix (its also an optional input too)